frontend/public/components/overview/networking-overview.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { Icon, ListGroup } from 'patternfly-react';
+
+import { K8sResourceKind } from '../../module/k8s';
+import { ResourceLink, SectionHeading } from '../utils';
+import { RouteLocation } from '../routes';
+
+export const ServicePortList: React.SFC<ServicePortListProps> = ({service}) => {
+  const ports = _.get(service, 'spec.ports', []);
+  return <ul className="port-list">
+    {
+      _.map(ports, ({name, port, protocol, targetPort}, i) =>
+        <li key={port || i}>
```

Comment:
Is `port` ever not defined?

Reply:
I don't think that's allowed, but we should check.

Note that `name` is unique if specified. We could do something like 

```tsx
<li key={name || `${protocol}/${port}`}>
```

Reply:
Yeah, just tried it, and it is allowed

Reply:
ok then I'd go with 

```jsx
<li key={name || `${protocol}/${port}`}>
```